lab-davidw/README.md

```diff
@@ -0,0 +1,13 @@
+#Lab 02: Tools and Context
```

Comment:
Nice job including a README. Generally speaking, a readme describes how to use your application.